cmd/minishift/cmd/start.go

```diff
@@ -198,14 +199,27 @@ func runStart(cmd *cobra.Command, args []string) {
 		atexit.Exit(1)
 	}
 
-	if err := openshift.AddSudoersRoleForUser("developer"); err != nil {
+	if err := oc.AddSudoerRoleForUser("developer"); err != nil {
 		glog.Errorln("Error adding developer user to sudoers", err)
 		atexit.Exit(1)
 	}
-	if err := openshift.AddContextForProfile(constants.MachineName, ip, "developer", "myproject"); err != nil {
+
+	if err := oc.AddContextForProfile(constants.MachineName, ip, "developer", "myproject"); err != nil {
 		glog.Errorln("Error adding OpenShift Context", err)
 		atexit.Exit(1)
 	}
+
+	// TODO - Issue #177 make sudoer and context part of addon as well
```

Comment:
Make sense to add sudoer and context part of addon . So these addons will be enabled by default right?